.gitignore

```diff
@@ -192,3 +192,18 @@ extension/bridge/target
 extension/bridge/vendor
 extension/bridge/.cargo_home
 extension/bridge/.cargo
+
+# Generated by Cargo
+# will have compiled files and executables
+debug/
+target/
+
+# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+# More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
+Cargo.lock
```

Comment:
I suspect we commit them. Can you check if we need to remove them?

Reply:
Interestingly, doesn't look like we commited them, no. https://github.com/mozilla-mobile/mozilla-vpn-client/tree/main/signature